_constitution/sse_2018_constitution.md

```diff
@@ -5,95 +5,128 @@ title: SSE Constitution
+Each elected officer shall hold their position for one year, starting in the Spring term. Yearly elections are to begin in late Winter term. Voting should occur the during the General Meeting during the 9th week of the Winter quarter. Election information should be distributed at least 2 weeks prior. Potential candidates must be nominated by at least two REGULAR MEMBERS (not including the candidate) prior to running for any executive position.
 ### Section 3: Officer Responsibilities
-* President
+* **All officers**
+    * Attend scheduled Executive Board Meetings
+        * Provide digital notice of absence in Slack channel at least 5 minutes before meeting
+        * An officer should not miss more than two consecutive meetings
+    * Should make an effort to attend General Meetings
+* **President**
     * Plan and lead General meetings
     * Plan and lead Executive meetings
     * Curate organization project opportunities
-* Vice President
+    * Work in tandem with the VP to achieve their responsibilities
+* **Vice President**
     * Serve as a liaison for University faculty and administration, industry representatives, and members of the local community
     * Curate job postings and career opportunities
-* Administrative Coordinator
+    * Work in tandem with the President to achieve their responsibilities 
+* **Administrative Coordinator**
     * Manage general (non-event related) fundraising efforts
     * Oversee the Organization’s account (balance and reimbursements)
     * Record minutes at General and Executive Meetings
-* Communications Coordinator
+    * Track regular membership point standings
+* **Communications Coordinator**
     * Serve as a liaison for leaders of other organizations on campus
-    * Publish periodic updates to all SE students
-    * Curate and distribute a bi-weekly (twice per month) newsletter with valuable information for members
+    * Publish periodic updates to all SE students via the website (and email for larger, more infrequent announcements)
+    * Curate and distribute a quarterly newsletter with valuable information for members and stakeholders
     * Forward high-impact updates from individual members to the general membership
-* Event Coordinator
+* **Event Coordinator**
     * Oversee / guide event committees to ensure events contribute to Organizational goals
-    * Manage SSE room and equipment reservations on AdAstra
-* Webmaster
-    * Manage SSE website discussions and content
+    * Manage SSE room and equipment reservations on the University’s room reservation software
+    * Refer to the event feedback forms from the previous year when planning new or recurring events
     * Maintain SSE event calendar
+* **Competitive Programming Team Coordinator**
+    * Actively recruit and develop members of the team
+    * Apply for competitions within the area, and if budget allows, in the US
```

Comment:
To make is clearer: "area" to "local area", "in the US" to "other areas in the US (i.e. Chicago)"